projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -19,7 +19,8 @@
 
 .t-flag {
     width: 1.75rem;
-    height: 1.25rem;
+    height: 1.75rem;
+    border-radius: 50%;
```

Comment:
You can already review this PR.
But don't merge. I am waiting for Sergey review

Reply:
Done